framework/source/class/qx/io/MDynamicScriptLoader.js

```diff
@@ -103,6 +103,25 @@ qx.Mixin.define("qx.io.MDynamicScriptLoader",
 
   /*
   *****************************************************************************
+     STATICS
+  *****************************************************************************
+  */
+
+  statics : {
+    /**
+     * a global hash with all scripts currently being processed 
+     */
+    __GLOBAL_LOADING : {},

```

Comment:
This would solve the "global" problem. Makes sense. I don't see, however, where __GLOBAL_LOADING[script] ever gets set to anything other than null. Have I just missed it someplace?